lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -315,6 +315,20 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     };
   }
 
+  void _reportAssetLoadError(String url,
+      ui.PlatformMessageResponseCallback? callback, String error) {
+    const MethodCodec codec = JSONMethodCodec();
+    final String message = 'Error while trying to load an asset $url';
+    if (!assertionsEnabled) {
+      /// For web/release mode log the load failure on console.
```

Comment:
Why is this necessary? Shouldn't the `_replyToPlatformMessage(...)` call below be enough to log the failure?